net/alien/inc/TAlienFile.h

```diff
@@ -31,9 +31,7 @@
 //                                                                      //
 //////////////////////////////////////////////////////////////////////////
 
-#ifndef ROOT_TXNetFile
```

Comment:
Why did you remove the fencing?

Reply:
I think this wasn't on the file in root/master...
I guess it doesn't hurt to add